.devcontainer/devcontainer.json

```diff
@@ -7,17 +7,19 @@
 	// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
 	"dockerFile": "Dockerfile",
 	// Set *default* container specific settings.json values on container create.
-	"settings": {
-		"terminal.integrated.shell.linux": null
+	"customizations": {
+		"vscode": {
+			"settings": {},
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"ms-python.python"
+			]
+		}
```

Comment:
Just making VSCode happy with the formatting. Also it doesn't seem to be happy with the shell part, but it doesn't seem like removal has done anything negative

Reply:
yea I've never noticed a benefit from having the shell setting in other `devcontainer.json` files either, glad to remove it!

Reply:
I think this is probably just a symptom of the old "copy and paste" and that probably persists in a lot of repos. Always great to simplify the  devcontainer.json file